pages/record/_.vue

```diff
@@ -88,6 +81,27 @@
             :europeana-identifier="identifier"
           />
         </div>
+        <div
+          class="card p-3 mb-3"
```

Comment:
Now there is always a card, even is there is no coreFields (or fields for the next one). This is an example where you have an empty card: en/record/2048012/Athena_Plus_ProvidedCHO_Openbaar_Kunstbezit_Vlaanderen_OKV1963_6

Reply:
I think this is out of scope for now. It would mean moving the language lookups up to the record page instead of containing it to the metadata field component. Also for an initial implementation this will highlight how metadata is/isn't displayed and we know that we'll need to refactor this once it goes through another iteration of design where we'll probably be very unlikely to end up with the situation where there is no metadata at all to show.